content/html/concepts/tags/terms/picture/picture.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '<picture>'
+Description: 'Represents multiple possible image sources to be applied to different devices and screen-sizes.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Images'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<picture>` tag represents multiple possible image sources to be applied to different devices and screen-sizes. While it has no semantic meaning, the `<picture>` tag will separate its contents from the rest of the document.
+
+## Syntax
+
+The `<picture>` tag is used with zero or more `<source>` elements followed by one [`<img>` element](https://www.codecademy.com/resources/docs/html/images).
+
+```pseudo
+<picture>
+  Zero or more <source> tags go here,
+  followed by at least one <image> tag here.
```

Comment:
```suggestion
  followed by at least one <img> tag here.
```

Reply:
Let's change this to `<img>` since we're referring to the image tag here rather than writing it out

Reply:
Since we've decided to refer to the tag itself, let's do so with the correct syntax 👍